packages/cli/src/Generate.ts

```diff
@@ -103,6 +104,7 @@ ${chalk.bold('Examples')}
       '--watch': Boolean,
       '--schema': String,
       '--data-proxy': Boolean,
+      '--deno': Boolean,
```

Comment:
Is there anyway to make Deno work without additional configuration? Mentioned before, but I'm hesitant to introduce another flavor of generated client.

Reply:
**Update:** just saw your comment in Slack

> I created another flavor (deno.js) because Deno Deploy doesn’t resolve CommonJS module (`require`). Does it make sense to change `edge.js` to use ES Modules instead of CommonJS? If so, I’ll try to make `edge.js` work both on Deno and other edge workers.
> BTW what are the supported workers of edge client? I think it supports cloudflare workers, but is there any other environment it should support?

We're currently targeting Node, Cloudflare and Vercel. Once we have this working, we can add Deno to our ecosystem tests: https://github.com/prisma/ecosystem-tests/tree/dev/dataproxy

Reply:
Ok. I'll remove '--deno' option as well. (What '--deno' option currently does is some validations about generator inputs, but that can be covered by the documentation)

> We're currently targeting Node, Cloudflare and Vercel. Once we have this working, we can add Deno to our ecosystem tests: https://github.com/prisma/ecosystem-tests/tree/dev/dataproxy

OK. I'll check this branch with those environments + deno.

Reply:
@kt3k FYI `require` should now work on Deploy, let's check if we can make Prisma work using it